src/Routing/Router.php

```diff
@@ -718,12 +718,12 @@ public static function extensions(array|string|null $extensions = null, bool $me
     {
         $collection = static::$_collection;
         if ($extensions === null) {
-            return array_unique(array_merge(static::$_defaultExtensions, $collection->getExtensions()));
+            return array_values(array_unique(array_merge(static::$_defaultExtensions, $collection->getExtensions())));
         }
 
         $extensions = (array)$extensions;
         if ($merge) {
-            $extensions = array_unique(array_merge(static::$_defaultExtensions, $extensions));
+            $extensions = array_values(array_unique(array_merge(static::$_defaultExtensions, $extensions)));
```

Comment:
we could think of maybe adding a method to the Hash Utility to do these 3 all at once with one call 😛 

We have those in quite a few places now.